test/cqlpy/test_protocol_exceptions.py

```diff
@@ -58,6 +59,45 @@ def try_connect(host, port, creds, protocol_version):
     with cql_with_protocol(host, port, creds, protocol_version) as session:
         return 1 if session else 0
 
+@pytest.fixture
+def debug_exceptions_logging(request, cql):
```

Comment:
I didn't notice this before I merged, so nevermind, but you don't need the "request" fixture here any more (the previous version of the patch needed it).